index.html

```diff
@@ -1733,18 +1733,59 @@ <h2>Subject is the Holder</h2>
 holder and all contained <a>verifiable credentials</a> are about a subject
 that can be identified to be the same as the holder.
       </p>
+     <p>
+If only the subject is allowed to present the verifiable credential to a
+verifier, the issuer MUST insert the "subject only" Terms of Use into the
+verifiable credential, as defined below. 
+
+<span class="issue">
+This feature is at risk and is likely to be removed due to lack of consensus.
+</span>
+      </p>
+
+   <section>
+
+   <h3> 'Subject Only' Terms of Use </h3>
+
+  <p>
+The Subject Only Terms of Use states that a verifiable credential MUST only be
+presented to a verifier by the subject. If a verifier is presented with a
+verifiable credential containing the Subject Only Terms of Use, by anyone other
+    than the subject, it MUST refuse to accept it.
```

Comment:
With selective disclosure, there is no way to ensure that a verifier ever sees the Subject Only Terms of Use.

Reply:
Please see the question here about this issue: https://github.com/w3c/vc-data-model/pull/282#issuecomment-440430172

Reply:
Post ZKP discussion: The selective disclosure schemes being implemented do indicate that a ToU is present in a credential definition, allowing for a verifier to request it (and thereby require it). The presenter can also prove if their particular credential happens to not possess a ToU should it be requested but not present.

Reply:
Great. So I will update the PR to remove conflicts with the latest version of the spec